ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -55,7 +56,7 @@ Rectangle {
 
     property var fileUrlsAndSources: []
 
-    property var imageErrorMessageLocation: StatusChatInput.ImageErrorMessageLocation.Top // TODO: Remove this proeprty?
+    property var imageErrorMessageLocation: StatusChatInput.ImageErrorMessageLocation.Top // TODO: Remove this property?
```

Comment:
👍

Reply:
Should I actually remove it? :)

Reply:
I chose not the best place to thumbsup a typo fix 😄 
No idea about the property, just thanks for all those minor fixes